microsoft-integration/impl/pom.xml

```diff
@@ -51,12 +51,14 @@
 
 		<!-- Microsoft dependencies -->
 		<dependency>
-			<groupId>com.azure</groupId>
-			<artifactId>azure-identity</artifactId>
```

Comment:
When I was reviewing libs I noticed that azure-identity sdk may not actually be needed as it only appears to be using msal4j.  Would need to test and make sure that this is accurate.

Reply:
We'll check it and come back to you once we ensure it keeps working like that, thanks!

Reply:
We've done some testing and everything seems to be working as expected with these changes, thanks!

Reply:
awesome thanks for the testing